### PR TITLE
feat: lists component

### PR DESCRIPTION
**Pull request by Sameony**
Added lists component
image
image

### CODE REVIEW

File: Styles/utilities.css [hunk 1]
```diff
@@ -38,6 +38,15 @@
 .flex-spc-arnd {
   justify-content: space-around;
 }
```
Comment (shuntzhere) [inline Styles/utilities.css:40]: Give it a meaningful name, this sounds confusing.

File: Styles/utilities.css [hunk 2]
```diff
@@ -38,6 +38,15 @@
...
+  justify-content: flex-end;
+}
+.flex-center {
+  justify-content: center;
+}
+.flex-start {
+  justify-content: flex-start;
+}
 .align-middle {
   align-items: center;
```
Comment (shuntzhere) [inline Styles/utilities.css:51]: Align-center would be a better name for this class.

File: Templates/Lists/lists.html [hunk 3]
```diff
@@ -168,7 +168,51 @@ <h1 class="pd-t-md"> Mirinda<span class="prim-color">UI</span> </h1>
...
+            <code class="showcase">
+        <ul class="flex flex-center">
+            <li class="list-item pd-sm mg-sm ">Item 1</li>
+            <li class="list-item pd-sm mg-sm ">Item 2</li>
+            <li class="list-item pd-sm mg-sm ">Item 3</li>
+        </ul>
+            </code>
+        </pre>
+
+    </div>
```
Comment (vaibhav-mahalle) [inline Templates/Lists/lists.html:215]: good use of utility classes. Nice and clean code.

**Review by vaibhav-mahalle — APPROVED** [review]

**Review by shuntzhere — APPROVED** [review]
Looks good, suggested a few changes. Do give them a look.

File: Styles/utilities.css (hunk 1 above)
Comment (Sameony) [reply Styles/utilities.css:40]: alright, i'll fix it after completing the current project. thanks for pointing it out

**Pull request merged**